core/state/event/event.go

```diff
@@ -29,10 +30,18 @@ const AppTopicState = "State change"
 
 // State represents the node state at the current moment. It's a read only object, used only to display data.
 type State struct {
-	NATStatus NATStatus        `json:"natStatus"`
-	Services  []ServiceInfo    `json:"serviceInfo"`
-	Sessions  []ServiceSession `json:"sessions"`
-	Consumer  ConsumerState    `json:"consumer"`
+	NATStatus  NATStatus        `json:"natStatus"`
+	Services   []ServiceInfo    `json:"serviceInfo"`
+	Sessions   []ServiceSession `json:"sessions"`
+	Consumer   ConsumerState    `json:"consumer"`
+	Identities []Identity       `json:"identities"`
+}
+
+// Identity represents identity and its status.
+type Identity struct {
+	Address            string                      `json:"id"`
```

Comment:
These JSON mapping not needed here, as you're remapping to `contract.IdentityDTO` later.
Is this intermediate structure needed at all?

Reply:
What are you suggesting to replace it with?